api/nodes/serializers.py

```diff
@@ -267,6 +267,18 @@ def create(self, validated_data):
             node.save()
         except ValidationValueError as e:
             raise InvalidModelValueError(detail=e.message)
+        if request.GET.get('inherit_contributors') and validated_data['parent'].has_permission(user, 'write'):
```

Comment:
Minor: `?inherit_contributors=false` would actually result in contributors getting inherited. You can fix this by using `api.base.utils.is_truthy`.